extensions/amp-ad/0.1/test/test-amp-ad-3p-impl.js

```diff
@@ -272,6 +272,7 @@ describe('amp-ad-3p-impl', () => {
         iframe.style.width = '300px';
         iframe.style.height = '200px';
         iframe.style.display = 'block';
+        iframe.style.minHeight = '0px';
```

Comment:
This iframe is created by ampAd, and has layout-fill. It's a test hack that we force to change its size to see if intersection box value was cached. Add this to override the new added `min-width: 100%; `